gossipsub.go

```diff
@@ -58,10 +65,15 @@ var (
 	GossipSubOpportunisticGraftPeers          = 2
 	GossipSubGraftFloodThreshold              = 10 * time.Second
 	GossipSubMaxIHaveLength                   = 5000
-	GossipSubMaxIHaveMessages                 = 10
+	GossipSubMaxIHaveMessages                 = 1000
```

Comment:
wait, why are we increasing the max IHAVE?

Reply:
It's a follow-up from this comment https://github.com/libp2p/go-libp2p-pubsub/pull/553#discussion_r1587434359

Reply:
Oh no, I intended to increase the GossipSubMaxIDontWantMessages. Sorry.